PLAN.md

```diff
@@ -1,8 +1,8 @@
-- Syntax highlighting
-    - Fix regression / change in highlighting
-    - Support document symbol
-    - Only update symbol if the LS supports it
+- Switching files between workspace
+    - reason-of-life - good example
 
+- Semicolon issue? Why is there always a semicolon error, even though examples compile fine without it?
```

Comment:
Thanks for looking at this, @jordwalke ! I've been going through the docs and I believe this was actually a misunderstanding on my part. I've been spending time jumping back and forth between OCaml/Reason syntax (learning OCaml primarily, and then playing with Reason) - I think this was a case where I didn't fully context switch between the two. I was caught up in some examples from BuckleScript w/o semicolons and was porting them over to Reason and missed some semicolons. Sorry for the noise, I don't believe there's a real issue here.